### PR TITLE
Added docker compose file for oc10, nC and ocis

### PR DESCRIPTION
**Pull request by Talank**
### OC10
- oc version: 10.4
- http_port: 8080
- Environment: Create database `owncloud` and grant all permission on that database to the user `owncloud` with password `owncloud`

### NC
- http_port: 8080
- Environment: Create database `nextcloud` and grant all permission on that database to the user `nextcloud` with password `nextcloud`

### CODE REVIEW

File: examples/oc10-compose/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      - redis
+    environment:
+      - OWNCLOUD_DOMAIN=localhost
+      - OWNCLOUD_DB_TYPE=mysql
+      - OWNCLOUD_DB_NAME=owncloud
+      - OWNCLOUD_DB_USERNAME=owncloud
+      - OWNCLOUD_DB_PASSWORD=owncloud
+      - OWNCLOUD_DB_HOST=db
+      - OWNCLOUD_ADMIN_USERNAME=owncloud
+      - OWNCLOUD_ADMIN_PASSWORD=owncloud
```
Comment (individual-it) [inline examples/oc10-compose/docker-compose.yml]: lets have these "admin:admin" to reduce confusion

File: examples/ocis-compose/config/identifier-registration.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (individual-it) [inline examples/ocis-compose/config/identifier-registration.yml]: how is this file used in the composer file?

File: examples/ocis-compose/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      KONNECTD_IDENTIFIER_REGISTRATION_CONF: /etc/ocis/identifier-registration.yml
+      KONNECTD_ISS: https://localhost:9200
+      KONNECTD_LOG_LEVEL: debug
+      KONNECTD_TLS: '0'
+      OCIS_LOG_LEVEL: debug
+      PHOENIX_OIDC_AUTHORITY: https://localhost:9200
+      PHOENIX_OIDC_METADATA_URL: https://localhost:9200/.well-known/openid-configuration
+      PHOENIX_WEB_CONFIG_SERVER: https://localhost:9200
+      PROXY_HTTP_ADDR: 0.0.0.0:9200
+      REVA_OIDC_ISSUER: https://localhost:9200
```
Comment (individual-it) [inline examples/ocis-compose/docker-compose.yml]: we need to be able to use the service through a public IP address, not only through localhost

e.g. you can use `${OCIS_DOMAIN}` like in https://github.com/owncloud-docker/compose-playground/blob/master/examples/eos-compose-acceptance-tests/docker-compose.yml#L54-L63

and the user will have to set the domain by:
`echo "OCIS_DOMAIN=1.2.3.4" > .env`

**Review by individual-it — CHANGES_REQUESTED** [review]
lets move the files/folders into a `benchmark` folder
e.g.:
```
benchmark
|__oc
     |__docker-compose.yml
|__nc
     |__docker-compose.yml
|__ocis
     |__docker-compose.yml

File: examples/ocis-compose/config/identifier-registration.yml (hunk 2 above)
Comment (Talank) [reply examples/ocis-compose/config/identifier-registration.yml]: it is used in `KONNECTD_IDENTIFIER_REGISTRATION_CONF: /etc/ocis/identifier-registration.yml` in the environment.

Comment (individual-it) [reply examples/ocis-compose/config/identifier-registration.yml]: I see, and `config` is used by the volumes

File: examples/benchmark/nc/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+      - db
+      - redis
+    environment:
+      - MYSQL_ROOT_PASSWORD=nextcloud
+      - MYSQL_HOST=db
+      - MYSQL_PASSWORD=nextcloud
+      - MYSQL_DATABASE=nextcloud
+      - MYSQL_USER=nextcloud
+      - NEXTCLOUD_ADMIN_USER=admin
+      - NEXTCLOUD_ADMIN_PASSWORD=admin
```
Comment (individual-it) [inline examples/benchmark/nc/docker-compose.yml:29]: also needs `NEXTCLOUD_TRUSTED_DOMAINS=myip`
should be dynamic like `OCIS_DOMAIN`

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**